azext_edge/edge/commands_edge.py

```diff
@@ -220,24 +220,52 @@ def create_instance(
         pass
 
 
-def upgrade(
+def upgrade_instance(
     cmd,
     resource_group_name: str,
     instance_name: str,
-    schema_registry_resource_id: Optional[str] = None,
     no_progress: Optional[bool] = None,
     confirm_yes: Optional[bool] = None,
+    ops_config: Optional[List[str]] = None,
```

Comment:
do we have a list of key value pairs that are actually accepted by the service? or do we assume that the service teams are the main consumers for this (and they would know the kv pairs themselves)

Reply:
Some user kvp's are described in AIO learn documentation intended for end users. Kvp's can be provided and used both internally and externally (power uses, troubleshooting etc) for whatever reasons.

Reply:
would be nice to include these in the long help but not necessary